main.py

```diff
@@ -2666,25 +2662,16 @@ async def callback_hell(interaction, thing):
                 pass
 
             if hidden_counter == 3 and user.dark_market_active:
-                await achemb(message, "dark_market", "followup")
                 if not user.story_complete:
                     # open the totally not suspicious dark market
                     await dark_market(message)
                 else:
                     await light_market(message)
-
+                await achemb(message, "dark_market", "followup")
+                
             if hidden_counter == 20:
                 await achemb(interaction, "darkest_market", "send")
 
-        # would be much more optimized but i cant get this to work
-        # for i in ["Cat Hunt", "Random", "Unfair"]:
-        #   if category == i:
-        #        buttons_list.append(Button(label=i, style=ButtonStyle.green))
-        #   else:
-        #        buttons_list.append(Button(label=i, style=ButtonStyle.blurple))
-        #   lambdas_list.append(lambda interaction : (await interaction.edit_original_response(embed=gen_new(i), view=insane_view_generator(i)) for _ in '_').__anext__())
-        #   buttons_list[-1].callback = lambdas_list[-1]
-
```

Comment:
probably should leave the comment there idk :silence:

Reply:
can i at least put my updated one that might work there instead ig